packages/core/src/render3/jit/module.ts

```diff
@@ -280,10 +287,8 @@ function verifySemanticsOfNgModuleDef(
     type = resolveForwardRef(type);
     const def = getComponentDef(type) || getDirectiveDef(type) || getPipeDef(type);
     if (def?.standalone) {
-      errors.push(`Unexpected "${stringifyForError(type)}" declaration in "${
-          stringifyForError(moduleType)}" NgModule. "${
-          stringifyForError(
-              type)}" is marked as standalone and can't be declared in any NgModule - did you intend to import it?`);
+      const location = `of the "${stringifyForError(moduleType)}" NgModule`;
```

Comment:
nit: since the `location` argument is required in the generator function, we can move the "of the" there as well, so it looks a bit cleaner on the caller side:
```suggestion
      const location = `"${stringifyForError(moduleType)}" NgModule`;
```